### PR TITLE
KOGITO-238 - Separate consumer for model and process instance

### PR DESCRIPTION
**Pull request by cristianonicolai**
Depends on https://github.com/kiegroup/kogito-bom/pull/53

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@ricardozanini we dont need to get rid of if then :)

File: data-index/data-index-service/src/main/java/org/kie/kogito/index/messaging/ReactiveMessagingEventConsumer.java [hunk 1]
```diff
@@ -29,19 +29,29 @@
 public class ReactiveMessagingEventConsumer {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ReactiveMessagingEventConsumer.class);
-    private static final String TOPIC = "kogito-processinstances-events";
-    
+    private static final String KOGITO_PROCESSINSTANCES_EVENTS = "kogito-processinstances-events";
+    private static final String KOGITO_PROCESSDOMAIN_EVENTS = "kogito-processdomain-events";
```
Comment (mswiderski) [inline data-index/data-index-service/src/main/java/org/kie/kogito/index/messaging/ReactiveMessagingEventConsumer.java]: is the expectation now to have two topics? I thought we will just have two consumers attached to the same topic as these will process the same messages

Comment (cristianonicolai) [reply data-index/data-index-service/src/main/java/org/kie/kogito/index/messaging/ReactiveMessagingEventConsumer.java]: it is using the same topic, as defined in the properties file. Somehow if I reuse the same name in the annotation one of the methods ends up not receiving any messages.

Comment (mswiderski) [reply data-index/data-index-service/src/main/java/org/kie/kogito/index/messaging/ReactiveMessagingEventConsumer.java]: Ah that’s perfect. Missed that

**Pull request merged**